src/pages/daemonEndpoint/GetDaemonEndpoint.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, ReactElement } from "react";
+import StringInput from "../operator/StringInput";
+
+// MaterialUI Imports
+import Card from "@material-ui/core/Card";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import { withStyles, createStyles } from "@material-ui/core/styles";
+import Divider from "@material-ui/core/Divider";
+
```

Comment:
Can we move all styles into its own file outside? Whats the best practise for this?